src/upgradesPage.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { FaArrowCircleLeft } from 'react-icons/fa';
+import ScrollMenu from 'react-horizontal-scrolling-menu';
+import './App.css';
+import {settings} from './constants'
+
+
+// list of skins
```

Comment:
Good work on implementing the tongue length upgrade. I would recommend deleting irrelevant comments and blank lines in this file